src/Resources/views/admin/order/update/_order_items.html.twig

```diff
@@ -28,7 +28,7 @@
                 {% set orderItem = itemForm.vars.data %}
                 <tr>
                     <td>{{ form_row(itemForm.quantity) }}</td>
-                    <td>{{ orderItem.productName }}</td> {# todo add variant #}
```

Comment:
The todo should not be removed, because we want to output the variant also, so people know exactly what product it is

Reply:
I can just fix it already 😄

Reply:
That's even better 🥇